client_encryption/field_level_encryption_config.py

```diff
@@ -24,19 +25,20 @@ def __init__(self, conf):
             self._paths[path] = EncryptionPathConfig(opt)
 
         if "encryptionCertificate" in json_config:
-            x509_cert = load_encryption_certificate(json_config["encryptionCertificate"])
-            self._encryption_certificate = dump_certificate(FILETYPE_ASN1, x509_cert)
+            x509_cert, cert_type = load_encryption_certificate(json_config["encryptionCertificate"])
+            self._encryption_certificate = x509_cert
+            #Fixed encoding is required, regardless of initial cerrtificate encoding to ensure correct calcualtion of fingerprint value
```

Comment:
`# Fixed encoding is required, regardless of initial certificate encoding to ensure correct calculation of fingerprint value`